<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionWindow</class>
 <widget class="QMainWindow" name="PartitionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Game</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GiB</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Region</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Partitions</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuConvert_Games_to">
     <property name="title">
      <string>Convert Games to...</string>
     </property>
    </widget>
    <addaction name="actionRefresh_List"/>
    <addaction name="actionTitles_From_WiiTDB"/>
    <addaction name="separator"/>
    <addaction name="actionPartition_Info"/>
    <addaction name="menuConvert_Games_to"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShit">
   <property name="text">
    <string>Shit</string>
   </property>
  </action>
  <action name="actionRefresh_List">
   <property name="text">
    <string>Refresh Game List</string>
   </property>
  </action>
  <action name="actionTitles_From_WiiTDB">
   <property name="text">
    <string>Titles From WiiTDB</string>
   </property>
  </action>
  <action name="actionPartition_Info">
   <property name="text">
    <string>Partition Info</string>
   </property>
  </action>
  <action name="action_wbfs">
   <property name="text">
    <string>.wbfs</string>
   </property>
  </action>
  <action name="action_iso">
   <property name="text">
    <string>.iso</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
